src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +29,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => setSelectTodo(undefined)}
```

Comment:
Don't use undefined as value, it's bad practice. If you want to pass empty value use `null`